site/data/i18n/en.json

```diff
@@ -186,13 +186,13 @@
 	"menu-footer-about-military-text": "Public service and military",
 	"menu-footer-about-military-link": "https://www.canada.ca/en/government/publicservice.html",
 	"menu-footer-about-news-text": "News",
-	"menu-footer-about-news-link": "http://www.servicecanada.gc.ca/gcnews",
+	"menu-footer-about-news-link": "https://www.canada.ca/en/news.html",
 	"menu-footer-about-treaties-text": "Treaties, laws and regulations",
 	"menu-footer-about-treaties-link": "https://www.canada.ca/en/government/system/laws.html",
 	"menu-footer-about-reporting-text": "Government-wide reporting",
 	"menu-footer-about-reporting-link": "https://www.canada.ca/en/transparency/reporting.html",
 	"menu-footer-about-pm-text": "Prime Minister",
-	"menu-footer-about-pm-link": "http://pm.gc.ca/eng",
+	"menu-footer-about-pm-link": "https://pm.gc.ca/eng",
```

Comment:
Although the splash page appears to support HTTPS, the language specific home pages do not